app/assets/javascripts/discourse/app/components/user-menu/menu.hbs

```diff
@@ -0,0 +1,43 @@
+<div class="user-menu revamped menu-panel drop-down" data-max-width="320">
+  <div class="panel-body">
+    <div class="panel-body-contents">
+      <div
+        id={{concat "quick-access-" this.currentTabId}}
+        class="quick-access-panel"
+        tabindex="-1"
+        aria-labelledby={{concat "user-menu-button-" this.currentTabId}}>
+        <this.currentPanelComponent/>
+      </div>
+      <div class="menu-tabs-container" role="tablist" aria-label={{i18n "user_menu.sr_menu_tabs"}}>
+        <div class="top-tabs tabs-list">
+          {{#each this.topTabs as |tab|}}
+            <UserMenu::Button
```

Comment:
I don't  think we need this component and can instead just use the native `<button>` element here.